src/configuration/formatting-evaluator-config.ts

```diff
@@ -18,7 +18,7 @@ const htmlType = Type.Record(Type.String(), Type.Number(), {
     code: 1,
     p: 1,
     em: 0,
-    img: 0,
+    img: 1,
```

Comment:
I'm assuming this was for testing. Let's default at 5 

As well as for code and links.

Reply:
This is not for testing, these are defaults if unspecified, will change them then.